smtp.yml

```diff
@@ -52,6 +52,21 @@ provision:
       type: string
       details: Email to recieve DMARC errors
       default: "datagovhelp@gsa.gov"
+    - field_name: create_sns_topics
+      type: boolean
+      details: Flag to toggle creation of SNS topics for feedback notifications
+      default: false
```

Comment:
I expect most people using this service may not have explicit AWS access to provision SNS topics on their own, and even if they do, they probably only care about specifying their own in the prod case. 

So, want to make the default here `true`?

Reply:
(...unless there's non-trivial cost/complexity associated SNS topics that we don't want to incur if they're not in use...)

Reply:
There's a decent complexity to actually getting the subscriptions to work, and I'm unclear on the cost for an SNS topic that's receiving the notifications but not draining, and also it's adding an additional AWS service, so I'd rather default to false